modules/ROOT/pages/deployment/services/s-list/thumbnails.adoc

```diff
@@ -15,6 +15,10 @@ Note for developers, more information is available with regards to querying thum
 * Thumbnails listens on port 9185 by default.
 * The default location storing thumbnails is $OCIS_BASE_DATA_PATH/thumbnails
 
+== Thumbnailing Performance
+
+Thumbnail generation can consume considerable resources. For thumbnailing, two libraries are available where one is embedded in the thumbnail service and one is external. The external shared library is not only much faster, but also provides more image types that can be converted. While the internal library is only available with the binary deployment, the external library is built and preconfigured with the container deployment only. Note, if required, you can manually add the shared library and build Infinite Scale for the binary deployment on your own using this library. To do so, see the https://owncloud.dev/services/thumbnails/[developer documentation, window=_blank] for more details.
```

Comment:
> For thumbnailing, two libraries are available where one is embedded in the thumbnail service and one is external.

Hm, it's a bit of hair-splitting about  IMO "external" is misleading or at least ambiguous. Both implementation rely on "external" libraries for image processing. ("external" as in using go-module that we didn't create ourselves) 

The main difference is that one implementation is written purely in go ( https://github.com/kovidgoyal/imaging) and allows us to create a statically linked executable with all dependencies included.

The other implementaion is based on an library written in C which is only available as a shared library and because of that requires dynamic linking, which means we cannot create a statically linked executable.

Reply:
I have updated the text to make it more precise without going into dev details.